code/modules/admin/verbs/deathsquad.dm

```diff
@@ -0,0 +1,197 @@
+//Deathsquad
+
+#define MAX_COMMANDOS 14
+
+GLOBAL_VAR_INIT(deathsquad_sent, FALSE)
+
+/client/proc/send_deathsquad()
+	var/client/proccaller = usr.client
+	if(!check_rights(R_EVENT))
+		return
+	if(SSticker.current_state == GAME_STATE_PREGAME)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
+		return
+	if(GLOB.deathsquad_sent)
+		if(alert("A Deathsquad is already being sent, are you sure you want to send another?",, "Yes", "No") != "Yes")
+			return
+	else
+		if(alert("Do you want to send in the Deathsquad? Once enabled, this is irreversible.",, "Yes", "No") != "Yes")
+			return
+	message_admins("<span class='notice'>[key_name_admin(proccaller)] has started to spawn a DeathSquad.</span>")
+	log_admin("[key_name_admin(proccaller)] has started to spawn a DeathSquad.")
+	alert("This 'mode' will go on until everyone is dead or the station is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. The first one selected/spawned will be the team leader.")
+
+	var/mission = sanitize(copytext(input(src, "Please specify which mission the Deathsquad shall undertake.", "Specify Mission", "",), 1, MAX_MESSAGE_LEN))
+	if(!mission)
+		if(alert("Error, no mission set. Do you want to exit the setup process?",, "Yes", "No") == "Yes")
+			message_admins("[key_name_admin(proccaller)] cancelled their Deathsquad.")
+			log_admin("[key_name(proccaller)] cancelled their Deathsquad.")
+			return
+
+	var/is_leader = TRUE // set to FALSE after leader is spawned
+	var/commando_number = clamp(input(src, "How many Deathsquad Commandos would you like to send? (Recommended is 6, Max is [MAX_COMMANDOS]])", "Specify Commandos") as num|null, 1, MAX_COMMANDOS)
+	if(!commando_number)
+		message_admins("[key_name_admin(proccaller)] cancelled their Deathsquad.")
+		log_admin("[key_name(proccaller)] cancelled their Deathsquad.")
+		return
+	if(GLOB.deathsquad_sent)
+		if(alert("A Deathsquad leader has previously been sent with an unrestricted NAD, would you like to spawn another?",, "Yes", "No") != "Yes")
+			is_leader = FALSE
+	GLOB.deathsquad_sent = TRUE
+	message_admins("[key_name_admin(proccaller)] has sent a Deathsquad with [commando_number] commandos.")
+	log_admin("[key_name(proccaller)] has sent a Deathsquad with [commando_number] commandos.")
+
+	// Find the nuclear auth code
+	var/nuke_code
+	for(var/obj/machinery/nuclearbomb/N in GLOB.machines)
+		var/temp_code = text2num(N.r_code)
+		if(temp_code)//if it's actually a number. It won't convert any non-numericals.
+			nuke_code = N.r_code
+			break
+
```

Comment:
```suggestion
	if(!nuke_code)
		message_admins("No nuclear warheads have been detected, the deathsquad will not be provided detonation codes.")

```
reccomend adding this, if no nuclear warheads are detected, there will not be codes in the spec ops manual. This will give needed feedback to admins so they can compensate by spawning in a nuke at CC.